Godeps/Godeps.json

```diff
@@ -3301,64 +3506,60 @@
 		},
 		{
 			"ImportPath": "google.golang.org/grpc",
-			"Comment": "v1.0.4",
-			"Rev": "777daa17ff9b5daef1cfdf915088a2ada3332bf0"
+			"Comment": "v1.0.2",
```

Comment:
The generated code is not compatible. Upstream kube uses 1.0.2. etcd uses 1.0.4.